includes/helper-functions.php

```diff
@@ -215,13 +213,10 @@ function edac_is_gutenberg_active() {
 function edac_days_active() {
 	$activation_date = get_option( 'edac_activation_date' );
 	if ( $activation_date ) {
-		$diff = strtotime( $activation_date ) - strtotime( gmdate( 'Y-m-d H:i:s' ) );
-
-		$days_active = abs( round( $diff / 86400 ) );
-	} else {
-		$days_active = null;
+		$diff        = strtotime( $activation_date ) - strtotime( gmdate( 'Y-m-d H:i:s' ) );
+		return abs( round( $diff / 86400 ) );
 	}
-	return $days_active;
+	return 0;
```

Comment:
This code was sometimes returning `null` instead of an integer, so that was fixed